smst_example_test.go

```diff
@@ -0,0 +1,48 @@
+package smt_test
```

Comment:
```suggestion
package smt
```

Reply:
No it needs to have the _test to act as an external application